Healthy/Classes/Modules/Login/LoginViewController.swift

```diff
@@ -34,6 +36,23 @@ final class LoginViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        if let token = AccessToken.current,
+           !token.isExpired {
+            // User is logged in, do work such as go to next view controller.
+            let token = token.tokenString
```

Comment:
Making the request is not a UI logic. This should be moved to FacebookAuthenticator. Please check GoogleAuthenticator for an example.